src/Spryker/Client/RabbitMq/Exception/QueuePoolsNotConfiguredException.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Increase `"spryker/store"` dependency to use new minor.

Reply:
i can not do it due to travis checks